bundles/ch.elexis.core/src/ch/elexis/core/types/LocalizeUtil.java

```diff
@@ -1,29 +1,69 @@
-package ch.elexis.core.types;
-
-import java.util.ResourceBundle;
-
-import org.eclipse.emf.common.util.Enumerator;
-
-import ch.elexis.core.interfaces.ILocalizedEnum;
-import ch.elexis.core.model.MaritalStatus;
-
-public class LocalizeUtil {
-	/**
-	 * Use with EMF generated not instances of {@link ILocalizedEnum}. Get the localized String for
-	 * the {@link Enumerator}.
-	 * 
-	 * @param enumerator
-	 * @return
-	 */
-	public static String getLocaleText(Enumerator enumerator){
-		if (enumerator != null) {
-			try {
-				return ResourceBundle.getBundle(ch.elexis.core.l10n.Messages.BUNDLE_NAME)
-					.getString(enumerator.getClass().getSimpleName() + "_" + enumerator.getName());
-			} catch (Exception e) {
-				return enumerator.getName();
-			}
-		}
-		return "?";
-	}
-}
+package ch.elexis.core.types;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import org.eclipse.emf.common.util.Enumerator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ch.elexis.core.interfaces.ILocalizedEnum;
+import ch.elexis.core.l10n.Messages;
+
+public class LocalizeUtil {
+	 private static Logger log = LoggerFactory.getLogger(LocalizeUtil.class);
```

Comment:
?! please implement ILocalizedEnum for the resp. interfaces, not a separate utility class